checkClock/app/assets/stylesheets/branches.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the branches controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```

Comment:
If you don't use this file, you can delete it.

Reply:
This file will be needed in the next issue. It will contain the styles for the branches views.